src/components/ProductList/ProductList.vue

```diff
@@ -0,0 +1,136 @@
+<template>
+
+  <ul class="items-container"> 
+    <ProductListItem v-for="item in items" 
+    :id="item.id" 
+    :key="item.id"
+    :title="item.title" 
+    :url="item.url"
+    :price="item.price"
+     @delete-item="openDeleteWindow"
+     @editing-item="openEditWindow"
+    />
+      <Modal 
+     v-if="showModal"
+    :initialTitle="item.title"
+    :initialPrice="item.price"
+    :initialUrl="item.url"
+    :itemId="item.id"
+    :title="action"
+    @close-modal="handleClose"
```

Comment:
Nitpick: Consider naming the event only `close` as it's clear that it is `modal` related.